sgt-back-end/index.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-console */
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    SELECT *
+      FROM "grades";
+  `;
+  db.query(sql)
+    .then(result => {
+      const table = result.rows;
+      res.json(table);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({
+        error: 'An unexpected error occured.'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res, next) => {
+  if (!req.body.name || !req.body.course || !req.body.score) {
+    res.status(400).json({
+      error: 'All fields are required.'
+    });
+    return;
+  }
+  if (req.body.score < 0 || req.body.score > 100) {
+    res.status(400).json({
+      error: 'Invalid score. Must be between 0 & 100.'
+    });
+    return;
+  }
+
+  const values = `
+    '${req.body.name}', '${req.body.course}', '${req.body.score}'
```

Comment:
the values should be in array format as in the example. This structure allows for sql injection attacks. It is best to stick with the ($1, $2, $3) format